docker/Dockerfile

```diff
@@ -6,6 +6,8 @@ ARG NUM_PROCESSES
 ENV NUM_PROCESSES ${NUM_PROCESSES:-4}
 ARG NUM_THREADS
 ENV NUM_THREADS ${NUM_THREADS:-2}
+ARG MAX_REQUESTS
+ENV MAX_REQUESTS ${MAX_REQUESTS:-0}
```

Comment:
is zero a good default? Does this mean there is no maximum?

Reply:
I'll remove this.